Community-Supported/clouddb-extractor/base_extractor.py

```diff
@@ -715,44 +679,50 @@ def query_to_hyper_files(
             raise Exception("Must specify either sql_query OR source_table")
 
         if source_table:
-            sql_query = "SELECT * from {}".format(source_table)
+            sql_query = "SELECT * from {0}{1}{2}".format(self.identifier_quote, source_table, self.identifier_quote)
```

Comment:
this quoting is still incorrect and allows for SQL injections...

Rather introduce a `quote_table_name` abstract functions which needs to be implemented by all the implementers of this base class. And the individual extractor implementations can then call into the functions provided by sqlalchemy/their native database driver and ensure that the table names are correctly quoted for the individual databases